install/kubernetes/helm/subcharts/mixer/templates/deployment.yaml

```diff
@@ -28,6 +28,7 @@
           - --monitoringPort={{ .Values.global.monitoringPort }}
           - --address
           - unix:///sock/mixer.socket
+          - --numCheckCacheEntries=0
```

Comment:
I don't think we want to add this here. We want to forcibly disable the cache, no matter what the user specifies. So I think we should just in the code and right at the point where the cache is being configure, we explicit pass 0 there and ignore what the user specifies on the command-line.